crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -115,7 +116,13 @@ public Mediator(Metadata md,
      * implementation can perform further validations.
      */
     @StopWatch(loggerName = "stopwatch.com.redhat.lightblue.mediator.Mediator", sizeCalculatorClass = "com.redhat.lightblue.mediator.ResponsePayloadSizeCalculator")
-    public Response insert(InsertionRequest req) {
+    public Response insert(InsertionRequest req, RequestMetrics metrics) {
+        RequestMetrics.Context metricCtx;
+        if (metrics.isBulkRequest()) {
+            metricCtx = metrics.startBulkRequest("insert", req.getEntityVersion().getEntity(), req.getEntityVersion().getVersion());
+        } else {
+            metricCtx = metrics.startEntityRequest("insert", req.getEntityVersion().getEntity(), req.getEntityVersion().getVersion());
```

Comment:
So the only reason I kept the measure related code for time being was because I was not sure how exactly the data collected through it is getting used. The mediator code suggests that we are just logging measure data and it's not used anywhere else, but I might be missing something. If it's clear that this is just logging and nothing else we can just remove measure altogether.